gh-pages/content/specification/6-compliance-report.md

```diff
@@ -0,0 +1,130 @@
+# Compliance Report
```

Comment:
Can you add a comment atop this file (`<!-- ... -->`) to denote it is automatically generated and should not be manually edited?

Reply:
Done